docker/pypi/wmagent-couchdb/Dockerfile

```diff
@@ -0,0 +1,100 @@
+ARG TAG=3.2.2
+FROM couchdb:${TAG}
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+ARG TAG
+ENV TAG=${TAG}
+RUN echo TAG=$TAG
+
+RUN apt-get update && apt-get install -y vim less sudo wget unzip python pip
+
+# # Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps emacs-nox tcpdump && apt-get clean
+
+RUN pip install CMSCouchapp
+
+# ENV USER=couchdb
+# ENV GROUP=couchdb
+ENV USER=cmst1
+ENV GROUP=zh
+ENV COUCH_UID=31961
+ENV COUCH_GID=1399
+ENV COUCH_PORT=5984
+# ENV COUCH_UID=5984
+# ENV COUCH_GID=5984
+
+ENV COUCH_ROOT_DIR=/data
+
+ENV COUCH_BASE_DIR=$COUCH_ROOT_DIR/srv/couchdb
+ENV COUCH_ADMIN_DIR=$COUCH_ROOT_DIR/admin/couchdb
+ENV WMA_ADMIN_DIR=$COUCH_ROOT_DIR/admin/wmagent
+ENV COUCH_CERTS_DIR=$COUCH_ROOT_DIR/certs
+
+ENV COUCH_CURRENT_DIR=$COUCH_BASE_DIR/$TAG
+ENV COUCH_MANAGE_DIR=$COUCH_CURRENT_DIR
+ENV COUCH_AUTH_DIR=$COUCH_BASE_DIR/auth/
+ENV COUCH_INSTALL_DIR=$COUCH_CURRENT_DIR/install
+ENV COUCH_STATE_DIR=$COUCH_CURRENT_DIR/state
+ENV COUCH_DATABASE_DIR=$COUCH_INSTALL_DIR/database
+ENV COUCH_CONFIG_DIR=$COUCH_CURRENT_DIR/config
+ENV COUCH_LOG_DIR=$COUCH_CURRENT_DIR/logs
+ENV COUCH_DEPLOY_DIR=/usr/local
+ENV COUCH_ENV_FILE=$COUCH_DEPLOY_DIR/deploy/env.sh
+ENV COUCH_SECRETS_FILE=$COUCH_ADMIN_DIR/CouchDB.secrets
```

Comment:
This comment is marked as resolved, but I see no follow up. It might be confusing to leave an unused secret reference in this docker file. I would suggest to remove this `COUCH_SECRETS_FILE` line?

Reply:
This was discussed with @khurtado in another comment/thread as well ( I cannot remember already which exactly), but we have agreed to keep the variable as a placeholder for the sake  of consistency  with the MariaDB container and because we do plan to fix this single user problem in the future.